apps/pie-microsite/src/assets/styles/components/_list.scss

```diff
@@ -0,0 +1,50 @@
+@use '../fozzie-config' as f;
+
+.c-list {
+    list-style: none;
+    padding-left: 0;
+}
+
+.c-list--ordered {
+    counter-reset: item;
```

Comment:
what is `item` here? I can't see it on the MDN docs

Reply:
As far as I understand, this is where we create a counter called 'item' then on line 12 we increase the counter by 1 and apply that number to the before.
I'll add a comment in
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Counter_Styles/Using_CSS_counters